### ISSUE TITLE
windows app 1.2.4

### ISSUE THREAD

**Issue by cmhdream** [gblazex/smoothscroll#217]
windows smoothscroll app version 1.2.4
https://www.smoothscroll.net/win/

Running the official website is not smooth
chrome smooth setting is turned off

Because it is the APP version, the website cannot be disabled
It is recommended to have a test page and adjust the settings.

**Comment by gblazex** [comment]
yes i went a little heavy on the effects. thanks for the feedback I'll try to fix it

**Comment by cmhdream** [comment]
working for me great new app.
https://www.desksoft.com/ScrollNavigator.htm

**Comment by Enlight432** [comment]
@gblazex 
When we use the Windows version, do we need to install an extension? Or should we not install the extension?
The Windows version also covers the browser

**Comment by Enlight432** [comment]
@gblazex 

When we use the Windows version, do we need to install an extension? Or should we not install the extension?
The Windows version also covers the browser
